src/v/kafka/server/handlers/incremental_alter_configs.cc

```diff
@@ -146,7 +146,10 @@ static void parse_and_set_topic_replication_factor(
   config_resource_operation op) {
     // set property value
     if (op == config_resource_operation::set) {
-        property.value = cluster::parsing_replication_factor(value);
+        property.value = std::nullopt;
+        if (value) {
```

Comment:
is this always `false` ?

Reply:
No
value is from request. So it can contains value if user provide it